scripts/entrypoint.pl

```diff
@@ -38,7 +38,7 @@
 sub cmd_httpd  {
     check_data_dir();
     wait_for_db();
-
+    run( '/usr/sbin/httpd', '-DFOREGROUND', '-f', '/app/httpd/httpd.conf' );
```

Comment:
As before, httpd seems fine. I guess checksetup isn't needed?

Reply:
Yep, looks like it